Frends.Oracle.ExecuteQuery/Frends.Oracle.ExecuteQuery.Tests/UnitTests.cs

```diff
@@ -0,0 +1,287 @@
+using NUnit.Framework;
+using System;
+using System.Threading;
+using System.Collections.Generic;
+using Frends.Oracle.ExecuteQuery.Definitions;
+using Newtonsoft.Json.Linq;
+
+namespace Frends.Oracle.ExecuteQuery.Tests
+{
+    /// <summary>
+    /// Docker container for Oracle database is needed to run these tests.
+    /// You need to have Docker installed before hand.
+    /// To create a docker container follow these instructions on Git CLI:
+    /// cd Frends.Oracle/Frends.Oracle.ExecuteQuery/build
+    /// ./deploy_oracle_docker_container.sh
+    /// Script might take a while (even 30 minutes) so relax.
+    /// To confirm that the container is up and running use Docker desktop or CLI. 
+    /// </summary>
+    [TestFixture]
+    [Ignore("Tests needs new Ubuntu based workflow")]
+    class TestClass
+    {
+        /// <summary>
+        /// Connection string for Oracle database.
+        /// </summary>
+        private static string _connectionString = "Data Source = (DESCRIPTION = (ADDRESS = (PROTOCOL = TCP)(HOST = localhost)(PORT = 51521))(CONNECT_DATA = (SERVICE_NAME = XEPDB1))); User Id = sys; Password=mysecurepassword; DBA PRIVILEGE=SYSDBA";
+        private static int _connectionTimeout = 30;
+
+        /// <summary>
+        /// Global variables.
+        /// </summary>
+        private static ConnectionProperties _connectionProperties;
+        private static QueryProperties _queryProperties;
+        private static Options _options;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _connectionProperties = new ConnectionProperties { ConnectionString = _connectionString, TimeoutSeconds = _connectionTimeout };
+            _queryProperties = new QueryProperties
+            {
+                Query = "create table workers(" +
+                "id NUMBER, " +
+                "first_name varchar2(50), " +
+                "last_name varchar2(50), " +
+                "start_date date, " +
+                "primary key(id))"
+            };
+            _options = new Options { ThrowErrorOnFailure = true, BindParameterByName = true };
+
+            Oracle.ExecuteQuery(_connectionProperties, _queryProperties, _options, new CancellationToken());
+        }
+
+        [TearDown]
+        public void TearDown()
```

Comment:
TearDown should also be done once after all tests are finished, not after every single test.

Reply:
Changed the SetUp and TearDown to OneTimeSetUp and OneTimeTearDown